pkg/detectors/satismeterprojectkey/satismeterprojectkey.go

```diff
@@ -40,49 +39,30 @@ func (s Scanner) Keywords() []string {
 func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
 	dataStr := string(data)
 
-	uniqueEmailMatches, uniqueKeyMatches, uniquePassMatches := make(map[string]struct{}), make(map[string]struct{}), make(map[string]struct{})
-	for _, match := range emailPat.FindAllStringSubmatch(dataStr, -1) {
-		uniqueEmailMatches[strings.TrimSpace(match[1])] = struct{}{}
+	uniqueProjectMatches, uniqueTokenMatches := make(map[string]struct{}), make(map[string]struct{})
+	for _, match := range projectPat.FindAllStringSubmatch(dataStr, -1) {
+		uniqueProjectMatches[strings.TrimSpace(match[1])] = struct{}{}
 	}
 
-	for _, match := range keyPat.FindAllStringSubmatch(dataStr, -1) {
-		uniqueKeyMatches[strings.TrimSpace(match[1])] = struct{}{}
+	for _, match := range tokenPat.FindAllStringSubmatch(dataStr, -1) {
+		uniqueTokenMatches[strings.TrimSpace(match[1])] = struct{}{}
 	}
 
-	for _, match := range passPat.FindAllStringSubmatch(dataStr, -1) {
-		uniquePassMatches[strings.TrimSpace(match[1])] = struct{}{}
-	}
+	for projectID := range uniqueProjectMatches {
+		for token := range uniqueTokenMatches {
+			s1 := detectors.Result{
+				DetectorType: detectorspb.DetectorType_SatismeterProjectkey,
+				Raw:          []byte(projectID),
+				RawV2:        []byte(projectID + token),
```

Comment:
Note: Although this change is technically breaking, we never recorded any verified credentials using the previous regex combination. Therefore, I believe it’s acceptable.